releases/mainnet/1.9.12/contracts.json

```diff
@@ -0,0 +1,297 @@
+    "MCD_JOIN_RWA002_A": "0xe72C7e90bc26c11d45dBeE736F0acf57fC5B7152",
+    "RWA002_A_URN": "0x225B3da5BE762Ee52B182157E67BeA0b31968163",
+    "RWA002_A_INPUT_CONDUIT": "0x2474F297214E5d96Ba4C81986A9F0e5C260f445D",
+    "RWA002_A_OUTPUT_CONDUIT": "0x2474F297214E5d96Ba4C81986A9F0e5C260f445D",
+    "RWA003": "0x07F0A80aD7AeB7BfB7f139EA71B3C8f7E17156B9",
+    "PIP_RWA003": "0xDeF7E88447F7D129420FC881B2a854ABB52B73B8",
+    "MCD_JOIN_RWA003_A": "0x1Fe789BBac5b141bdD795A3Bc5E12Af29dDB4b86",
+    "RWA003_A_URN": "0x7bF825718e7C388c3be16CFe9982539A7455540F",
+    "RWA003_A_INPUT_CONDUIT": "0x2A9798c6F165B6D60Cfb923Fe5BFD6f338695D9B",
+    "RWA003_A_OUTPUT_CONDUIT": "0x2A9798c6F165B6D60Cfb923Fe5BFD6f338695D9B",
+    "RWA004": "0x873F2101047A62F84456E3B2B13df2287925D3F9",
+    "PIP_RWA004": "0x5eEE1F3d14850332A75324514CcbD2DBC8Bbc566",
+    "MCD_JOIN_RWA004_A": "0xD50a8e9369140539D1c2D113c4dC1e659c6242eB",
+    "RWA004_A_URN": "0xeF1699548717aa4Cf47aD738316280b56814C821",
+    "RWA004_A_INPUT_CONDUIT": "0xe1ed3F588A98bF8a3744f4BF74Fd8540e81AdE3f",
+    "RWA004_A_OUTPUT_CONDUIT": "0xe1ed3F588A98bF8a3744f4BF74Fd8540e81AdE3f",
+    "RWA005": "0x6DB236515E90fC831D146f5829407746EDdc5296",
+    "PIP_RWA005": "0x8E6039C558738eb136833aB50271ae065c700d2B",
+    "MCD_JOIN_RWA005_A": "0xA4fD373b93aD8e054970A3d6cd4Fd4C31D08192e",
+    "RWA005_A_URN": "0xc40907545C57dB30F01a1c2acB242C7c7ACB2B90",
+    "RWA005_A_INPUT_CONDUIT": "0x5b702e1fEF3F556cbe219eE697D7f170A236cc66",
+    "RWA005_A_OUTPUT_CONDUIT": "0x5b702e1fEF3F556cbe219eE697D7f170A236cc66",
+    "RWA006": "0x4EE03cfBF6E784c462839f5954d60f7C2B60b113",
+    "PIP_RWA006": "0xB8AeCF04Fdf22Ef6C0c6b6536896e1F2870C41D3",
+    "MCD_JOIN_RWA006_A": "0x5E11E34b6745FeBa9449Ae53c185413d6EdC66BE",
+    "RWA006_A_URN": "0x0C185bf5388DdfDB288F4D875265d456D18FD9Cb",
+    "RWA006_A_INPUT_CONDUIT": "0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3",
+    "RWA006_A_OUTPUT_CONDUIT": "0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3",
+    "PROXY_PAUSE_ACTIONS": "0x6bda13D43B7EDd6CAfE1f70fB98b5d40f61A1370",
+    "PROXY_DEPLOYER": "0x1b93556AB8dcCEF01Cd7823C617a6d340f53Fb58",
+    "OPTIMISM_DAI_BRIDGE": "0x10E6593CDda8c58a1d0f14C5164B376352a55f2F",
+    "OPTIMISM_ESCROW": "0x467194771dAe2967Aef3ECbEDD3Bf9a310C76C65",
+    "OPTIMISM_GOV_RELAY": "0x09B354CDA89203BB7B3131CC728dFa06ab09Ae2F",
+    "ARBITRUM_DAI_BRIDGE": "0xD3B5b60020504bc3489D6949d545893982BA3011",
+    "ARBITRUM_ESCROW": "0xA10c7CE4b876998858b1a9E12b10092229539400",
+    "ARBITRUM_GOV_RELAY": "0x9ba25c289e351779E0D481Ba37489317c34A899d",
+    "GUNIV3DAIUSDC2": "0x50379f632ca68D36E50cfBC8F78fe16bd1499d1e",
+    "PIP_GUNIV3DAIUSDC2": "0xcCBa43231aC6eceBd1278B90c3a44711a00F4e93",
+    "MCD_JOIN_GUNIV3DAIUSDC2_A": "0xA7e4dDde3cBcEf122851A7C8F7A55f23c0Daf335",
+    "MCD_CLIP_GUNIV3DAIUSDC2_A": "0xB55da3d3100C4eBF9De755b6DdC24BF209f6cc06",
+    "MCD_CLIP_CALC_GUNIV3DAIUSDC2_A": "0xef051Ca2A2d809ba47ee0FC8caaEd06E3D832225"
+  
+  }
+  
```

Comment:
extra line - eof missing newline

Reply:
some spacing as well ([diffcheck](https://www.diffchecker.com/l7fD34QT))

Reply:
we should hash it again reflecting the changes and update the `index.html` as well